Server/Include/ClientHandler.cpp

```diff
@@ -50,12 +56,31 @@ void ClientHandler::OnPacketReceived(LambdaEngine::IClient* pClient, LambdaEngin
 	LOG_MESSAGE("OnPacketReceived()");
 
 	LambdaEngine::BinaryDecoder decoder(pPacket);
-	std::string name = decoder.ReadString();
-	LOG_MESSAGE(name.c_str());
+	if (counter + 1 != decoder.ReadUInt32())
+	{
+		//DEBUGBREAK();
+	}
+	counter++;
+	LOG_MESSAGE("%d", (int)counter);
 }
 
 void ClientHandler::OnClientReleased(LambdaEngine::IClient* pClient)
 {
 	UNREFERENCED_VARIABLE(pClient);
 	delete this;
 }
+
+void ClientHandler::OnPacketDelivered(LambdaEngine::NetworkSegment* pPacket)
+{
+	LOG_ERROR("OnPacketDelivered(%s)", pPacket->ToString().c_str());
+}
+
+void ClientHandler::OnPacketResent(LambdaEngine::NetworkSegment* pPacket, uint8 retries)
+{
+
+}
+
+void ClientHandler::OnPacketMaxTriesReached(LambdaEngine::NetworkSegment* pPacket, uint8 retries)
+{
+
```

Comment:
To avoid linter warnings in these function, use UNREFERENCED_VARIABLE as done on other functions
```suggestion
	UNREFERENCED_VARIABLE(pPacket);
	UNREFERENCED_VARIABLE(retries);
```